site/docs/concepts/architecture.md

```diff
@@ -3,31 +3,26 @@
 Fybrik takes a modular approach to provide an open platform for controlling
 and securing the use of data across an organization. The figure below showcases the
 current architecture of the Fybrik platform, running on top of Kubernetes. 
-The storage systems shown in the lower half of the figure are merely an example.
 
 The core parts of Fybrik are based on Kubernetes controllers and Custom Resource Definitions (CRDs) in order to define its work items and reconcile the state of them.
 
-The primary interaction object for a data user is the `FybrikApplication` custom resource where a user defines which data should be used for which purpose. The following chart and description describe the architecture and components of Fybrik relative to when they are used.
+The primary interaction object for a data user is the `FybrikApplication` custom resource (1) where a user defines which data should be used for which purpose. The following chart and description describe the architecture and components of Fybrik relative to when they are used.
 
-![Architecture](../static/workflow_multicluster.svg)
+![Architecture](../static/architecture.png)
 
 Before the data user can perform any actions a data operator has to [install](../get-started/quickstart.md) Fybrik and modules. 
-[Modules](./modules.md) describe capabilities that can be included in a data plane.  These may be existing open source or third party service, or custom ones.  The module of a service indicates the capabilities it supports, the formats and interfaces, and how to deploy the service.  Modules may describe externally deployed services, or services deployed by fybrik.  Examples of modules are those that provide read/write access or produce implicit copies that serve as lower latency caches of remote assets. Modules also may also perform actions usesd to enforce data governance policy decisions, such as masking or redaction as examples.
+[Modules](./modules.md) (M) describe capabilities that can be included in a data plane.  These may be existing open source or third party service, or custom ones.  The module of a service indicates the capabilities it supports, the formats and interfaces, and how to deploy the service.  Modules may describe externally deployed services, or services deployed by fybrik.  Examples of modules are those that provide read/write access or produce implicit copies that serve as lower latency caches of remote assets. Modules may also perform actions to enforce data governance policy decisions, such as masking or redaction as examples.
```

Comment:
"third party service, or customer ones."  services - should be plural

Reply:
fixed